policyengine_us/parameters/gov/states/ca/cdss/child_care/eligibility/welfare_to_work.yaml

```diff
@@ -0,0 +1,11 @@
+description: California CalWORKs Child Care participants should be engaged in a county-approved Welfare-to-Work (WtW).
```

Comment:
what is the purpose of this file? If it is just this one variable then we can just the variable directly in the variable